unittests/tools/test_wpscan_parser.py

```diff
@@ -74,18 +74,24 @@ def test_parse_file_with_multiple_vuln(self):
             for endpoint in finding.unsaved_endpoints:
                 endpoint.clean()
         self.assertEqual(6, len(findings))
-        finding = findings[0]
-        self.assertEqual("7391118e-eef5-4ff8-a8ea-f6b65f442c63", finding.unique_id_from_tool)
-        self.assertNotEqual("Info", finding.severity)  # it is a vulnerability so not 'Info'
-        self.assertEqual("Contact Form 7 < 5.3.2 - Unrestricted File Upload", finding.title)
-        self.assertEqual("CVE-2020-35489", finding.cve)
-        self.assertEqual(datetime.datetime(2021, 3, 17, 12, 21, 6), finding.date)
-        #
-        finding = findings[4]
-        self.assertIsNone(finding.unique_id_from_tool)  # interesting findings are not vlunerability
-        self.assertEqual("Info", finding.severity)  # it is not a vulnerability so severity should be 'Info'
-        self.assertEqual("Interesting finding: WordPress readme found: http://example/readme.html", finding.title)
-        self.assertEqual(datetime.datetime(2021, 3, 17, 12, 21, 6), finding.date)
+
+        with self.subTest(i=0):
+            finding = findings[0]
+            self.assertEqual("7391118e-eef5-4ff8-a8ea-f6b65f442c63", finding.unique_id_from_tool)
+            self.assertNotEqual("Info", finding.severity)  # it is a vulnerability so not 'Info'
+            self.assertEqual("Contact Form 7 < 5.3.2 - Unrestricted File Upload", finding.title)
+            self.assertEqual("CVE-2020-35489", finding.cve)
+            self.assertEqual(datetime.datetime(2021, 3, 17, 12, 21, 6), finding.date)
+            self.assertEqual("Certain", finding.get_scanner_confidence_text())  # data are => 100%
```

Comment:
This finding has no confidence, so the scanner confidence text should not be `Certain`

Reply:
fixed it